sci-misc/boinc/boinc-7.2.0-r1.ebuild

```diff
@@ -23,7 +23,7 @@ RDEPEND="
 	!sci-misc/boinc-bin
 	!app-admin/quickswitch
 	>=app-misc/ca-certificates-20080809
-	dev-libs/openssl
+	dev-libs/openssl:=
```

Comment:
@prometheanfire That was my first idea, but since @mgorny commented in the first line note ("Does boinc use openssl at build time? If so, it needs OpenSSL headers and :* is invalid then since other slots don't provide the headers.") I changed it to `:=`.
Anyway, now I changed it once again to `:0=`. I still need to get deeper into slotting..